Bottom: 27a88d4421a75c3d537b316cc2203d1e75b1a980
Top:    318d585ee6c067c88511329a327192879a086cee
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 14:51:00 +0100

Refresh of bentley-ottann-test-implementa

---

diff --git a/src/cairo/cairo-bentley-ottmann.c b/src/borast/cairo-bentley-ottmann.c
similarity index 100%
rename from src/cairo/cairo-bentley-ottmann.c
rename to src/borast/cairo-bentley-ottmann.c
diff --git a/src/cairo/cairo-combsort-private.h b/src/borast/cairo-combsort-private.h
similarity index 100%
rename from src/cairo/cairo-combsort-private.h
rename to src/borast/cairo-combsort-private.h
diff --git a/src/cairo/cairo-compiler-private.h b/src/borast/cairo-compiler-private.h
similarity index 100%
rename from src/cairo/cairo-compiler-private.h
rename to src/borast/cairo-compiler-private.h
diff --git a/src/cairo/cairo-fixed-private.h b/src/borast/cairo-fixed-private.h
similarity index 100%
rename from src/cairo/cairo-fixed-private.h
rename to src/borast/cairo-fixed-private.h
diff --git a/src/cairo/cairo-fixed-type-private.h b/src/borast/cairo-fixed-type-private.h
similarity index 100%
rename from src/cairo/cairo-fixed-type-private.h
rename to src/borast/cairo-fixed-type-private.h
diff --git a/src/cairo/cairo-freelist-private.h b/src/borast/cairo-freelist-private.h
similarity index 100%
rename from src/cairo/cairo-freelist-private.h
rename to src/borast/cairo-freelist-private.h
diff --git a/src/cairo/cairo-freelist.c b/src/borast/cairo-freelist.c
similarity index 100%
rename from src/cairo/cairo-freelist.c
rename to src/borast/cairo-freelist.c
diff --git a/src/cairo/cairo-malloc-private.h b/src/borast/cairo-malloc-private.h
similarity index 100%
rename from src/cairo/cairo-malloc-private.h
rename to src/borast/cairo-malloc-private.h
diff --git a/src/cairo/cairo-minimal.h b/src/borast/cairo-minimal.h
similarity index 100%
rename from src/cairo/cairo-minimal.h
rename to src/borast/cairo-minimal.h
diff --git a/src/cairo/cairo-traps-private.h b/src/borast/cairo-traps-private.h
similarity index 100%
rename from src/cairo/cairo-traps-private.h
rename to src/borast/cairo-traps-private.h
diff --git a/src/cairo/cairo-traps.c b/src/borast/cairo-traps.c
similarity index 100%
rename from src/cairo/cairo-traps.c
rename to src/borast/cairo-traps.c
diff --git a/src/cairo/cairo-types-private.h b/src/borast/cairo-types-private.h
similarity index 100%
rename from src/cairo/cairo-types-private.h
rename to src/borast/cairo-types-private.h
diff --git a/src/cairo/cairo-wideint-private.h b/src/borast/cairo-wideint-private.h
similarity index 100%
rename from src/cairo/cairo-wideint-private.h
rename to src/borast/cairo-wideint-private.h
diff --git a/src/cairo/cairo-wideint-type-private.h b/src/borast/cairo-wideint-type-private.h
similarity index 100%
rename from src/cairo/cairo-wideint-type-private.h
rename to src/borast/cairo-wideint-type-private.h
diff --git a/src/cairo/cairo-wideint.c b/src/borast/cairo-wideint.c
similarity index 100%
rename from src/cairo/cairo-wideint.c
rename to src/borast/cairo-wideint.c
diff --git a/src/cairo/cairoint-minimal.h b/src/borast/cairoint-minimal.h
similarity index 100%
rename from src/cairo/cairoint-minimal.h
rename to src/borast/cairoint-minimal.h
